src/platforms/common/profiling/index.mdx

```diff
@@ -108,6 +133,32 @@ The `io.sentry.traces.profiling.sample-rate` setting is *relative* to the `io.se
 
 </PlatformSection>
 
+<PlatformSection supported={["python"]}>
+
+<Note>
+
+Python profiling alpha is available starting in SDK version `1.11.0` and only for WSGI applications.
```

Comment:
@lizokm How does this sound to you? We want to be clear that the minimum SDK version required and the tyes of applications this will work with out of the box.

Reply:
This sounds good to me. Apologies for the delay :)